unlock-protocol.com/blog/ejecting-account.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ejecting an Unlock Account
+subTitle: Graduating with grace!
+authorName: Julien Genestoux
+publishDate: Nov 5, 2019
+description: Unlock accounts are regular Ethereum accounts. You can easily take it over and use it with your own web3 wallet!
+image: /static/images/blog/ejecting-account/ejecting.jpg
+---
+
+![Eject Unlock Account](/static/images/blog/ejecting-account/ejecting.jpg)
+
+A couple of weeks ago, we introduced the [Unlock User Accounts](/blog/unlock-user-accounts). These accounts behave exactly like any other web account, with an email and password. Yet, under the hood, they're actually powerful Ethereum wallets with a private key which could be used to send transactions, receive funds, or interract directly with any smart contract.
+
+In the Ethereum world, an account is identified by a pair of cryptographic keys. The public key, also known as the user's address, is a way to **identify** someone, while the private key is used to **authenticate** that user. The cryptographic keys are "self-sovereign" which means that they do not require any 3rd party to be created, or to be used, unlike a regular account on a web applications, which requires this web application to exist. This is a very powerful mechanism which guarantees a permissionless usage.
+
+Unfortunately, this comes with a set of disadvantages. First: these keys are nearly impossible to remember, or even read and type! Their permissionlessness aspect also means that if someone forgets (or loses) them, no-one can help them... and they may be locked out of their account forever. Since the web was not built with such a system at its core, our muscle memory is not built around these concepts, which means that they're intimidating and foreign, but also require the use of tools that we're not familiar with, such as crypto wallets or extensions.
```

Comment:
```suggestion
Unfortunately, this comes with a set of disadvantages. First: these keys are nearly impossible to remember, or even read and type! Their permissionlessness aspect also means that if someone forgets (or loses) them, no-one can help them... and they may be locked out of their account forever. Since the web was not built with such a system at its core, our muscle memory is not built around these concepts, which means that they're intimidating and foreign, but also require the use of tools that we're not familiar with, such as crypto wallets or browser extensions.
```